samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -165,6 +176,17 @@ static void button_handler_cb(uint32_t pressed, uint32_t changed)
 			printk("Error getting sensor descriptor (%d)\n", err);
 		}
 	}
+	if (pressed & BIT(3)) {
```

Comment:
Not really related to your PR, but this will fire if I hold down button 3 while pressing a different button, since this cb fires because of the unrelated press and because it's held down `pressed & BIT(3)` is `true`. All of these should be `if(changed & pressed & BIT(n))`.

Reply:
I assumed that's a feature. One can do parallel button presses.

Reply:
No, you misunderstand. The way it works now, every button which is down will trigger the routine once _any_ of them changes. If three buttons are held down, all four messages will be sent when I press or relase the final one.